node/src/chain_spec/turing.rs

```diff
@@ -16,7 +16,7 @@ use turing_runtime::{
 
 static TOKEN_SYMBOL: &str = "TUR";
 const SS_58_FORMAT: u32 = 51;
-const TOTAL_TOKENS: u128 = DOLLAR * 1_000_000_000;
+const TOTAL_TOKENS: u128 = DOLLAR * 58_000_000;
```

Comment:
variable currently called TOTAL_TOKENS. wonder if i should rename to be INITIAL_ALLOCATION instead?

Reply:
let's not have a global for this one. I'd set it specifically w/ i the functions